modules/fts/pages/fts-pagination-skip-limit.adoc

```diff
@@ -0,0 +1,41 @@
+= skip and limit
```

Comment:
What's this section about? There's no such settings.

You've already covered all the supported pagination techniques in the 2 previous files.